src/Persistence/Reflection/RuntimePublicReflectionProperty.php

```diff
@@ -49,6 +53,13 @@ public function setValue($object, $value = null)
             return;
         }
 
+        if (! method_exists($object, '__getInitializer') || ! method_exists($object, '__setInitializer')) {
```

Comment:
Couldn't we assume that $object is always an isntance of the class used for this property and move the check to ctor?

Reply:
Ah yes that's probably safe to do 👍